hierarchicalforecast/methods.py

```diff
@@ -30,20 +30,46 @@ def bottom_up(S: np.ndarray,
     W = np.eye(n_hiers, dtype=np.float32)
     return _reconcile(S, P, W, y_hat)
 
-# %% ../nbs/methods.ipynb 6
+# %% ../nbs/methods.ipynb 7
 class BottomUp:
+    """Bottom Up Reconciliation Class.
+    [Source code](https://github.com/dluuo/hierarchicalforecast/blob/main/hierarchicalforecast/methods.py).
+
```

Comment:
Let's make the latex code compile in the documentation.
Try to use `\\` because `\` is a Markdown restricted character.